keps/prod-readiness/sig-storage/596.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 596
+stable:
+  approver: "@deads2k"
```

Comment:
@deads2k - if you want to reshuffle, I can take this.

Reply:
@wojtek-t I'll put you on the spreadsheet, I suggest you take this one

Reply:
@dobsonj - please reassign to me

Reply:
Updated